### ISSUE TITLE
reproduce similar results

### ISSUE THREAD

**Issue by lolongcovas** [jay-mahadeokar/pynetbuilder#9]
#reproduce similar results
Hi, I am trying to reproduce similar results of imageNet classification task using resnet18, 36 and 50. The proto files were generated by:

```
python app/imagenet/build_resnet.py -m bottleneck -b 2 2 2 2 -n 64 --no-fc_layers -o /tmp/
```
which is bottleneck version of resnet18. the image preprocessing is, basically, random crops and flips (no pca color augmentation described in the paper). The learning strategy was lr=0.01, stepsize of 150K iters with gamma 0.1, momentum 0.9 and weight decay 0.0001. the final top-1 accuracy on validation set I got 49.58%. However, in the paper they achieve 72.12% (resnet18). There is a huge gap between our model and their paper's reported. Do you know why?

Also, your resnet50 model scores 71.75% however, the original paper scores 77.15. There are 5point of difference. Do you think color augmentation and batch shuffle for each epoch affect this gap?

Best

**Comment by KevinYuk** [comment]
@lolongcovas How about resnet18 mAP?

**Comment by KevinYuk** [comment]
@lolongcovas Hi,
What's your resnet18 bottleneck version accuracy now?
It seems that the lr should begin from 0.1, not 0.01.
